cmd/ko/tags.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type TagsOptions struct {
+	Tags []string
+}
+
+func addTagsArg(cmd *cobra.Command, ta *TagsOptions) {
+	cmd.Flags().StringSliceVar(&ta.Tags, "tags", ta.Tags,
```

Comment:
I think i'd like this to be `StringSliceVarP` with `-t` as the shorthand so we can use it similarly to `docker build -t foo -t bar`. @ImJasonH @mattmoor thoughts?

Reply:
that actually seems cool and very useful. I've added this too.